packages/bot-web-ui/src/components/dashboard/quick-strategy/quick-strategy-components/inputs.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import classNames from 'classnames';
+import { InputField } from '.';
+import { TInputsFieldNames } from '../quick-strategy.types';
+import { TInputs } from './components.types';
+
+const Inputs = ({
+    idx,
+    is_mobile,
+    field_name,
+    id,
+    className,
+    label,
+    input_value,
+    placeholder,
+    is_uniq_strategy_field,
+    trailing_icon_message,
+    zIndex,
+    uniq_selected_input,
+    errors,
+    is_input_field,
+    handleChange,
+    onChangeInputValue,
+    setCurrentFocus,
+}: TInputs) =>
+    is_input_field ? (
```

Comment:
same here

Reply:
@rupato-deriv answer will be the same as above